dev/tools/vitool/pubspec.yaml

```diff
@@ -10,19 +10,21 @@ environment:
 dependencies:
   args: 1.6.0
   vector_math: 2.1.0-nullsafety.5
-  xml: 5.0.0-nullsafety.1
+  xml: 4.5.1
```

Comment:
I think we'd want to keep these on nullsafe versions in general - it might require pinning that for now in the upgrade_packages.dart though.

Reply:
Any idea why it fell back in the first place? I didn't change anything, just reran the tool like normal.

Reply:
I believe it's because nullsafe packages are prerelease and we no longer have a pinned dep or direct dep that requires them

Reply:
You mean package:image was a load-bearing dependency?

Reply:
I guess? We basically just build one giant pubspec with all the packages we use and try to resolve them all to latest versions. I know for some others to pick up null safety I had to update the pinned list, presumably because pub tries to resolve latest release versions rather than latest pre-releases.